Azure.ResourceManager.Core.Tests/Azure.ResourceManager.Core.Tests.csproj

```diff
@@ -16,12 +16,12 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
   </ItemGroup>
-  <Target Name="GenerateHtmlCoverageReport" AfterTargets="GenerateCoverageResultAfterTest">
+  <!--Target Name="GenerateHtmlCoverageReport" AfterTargets="GenerateCoverageResultAfterTest">
```

Comment:
doesn't this just comment it out?  Does the html code coverage report still get generated?